packages/strapi-plugin-content-manager/services/ContentManager.js

```diff
@@ -83,17 +83,34 @@ module.exports = {
     return strapi.entityService.countSearch({ params: query }, { model });
   },
 
-  publish(params, model) {
-    return strapi.entityService.update(
+  async publish(params, model) {
+    const modelDef = strapi.getModel(model);
+
+    const publishedEntry = await strapi.entityService.update(
       { params, data: { [PUBLISHED_AT_ATTRIBUTE]: new Date().toISOString() } },
       { model }
     );
+
+    strapi.eventHub.emit('entry.publish', {
```

Comment:
Little side-note: Shouldn't we have something like an enum used to store all the literal values of the webhooks? As the app (and the amount of webhook) grows, we (IMO) should consider grouping those kinds of values together to make contributions easier.
What do you think?